hack/update-bazel.sh

```diff
@@ -23,4 +23,15 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 git config http.https://gopkg.in.followRedirects true
 
 go get -u gopkg.in/mikedanese/gazel.v14/gazel
-"${GOPATH}/bin/gazel" -root="$(kube::realpath ${KUBE_ROOT})"
+
+for path in ${GOPATH//:/ }; do
+    if [ -e "${path}/bin/gazel" ]; then
```

Comment:
nit: I think we prefer `[[ ]]`?
https://google.github.io/styleguide/shell.xml#Test,_%5B_and_%5B%5B